dist/style-guide/item-general-skip-links.html

```diff
@@ -119,12 +119,12 @@ <h2 class="kss-title kss-title--level-2">
 
                             <div class="kss-description">
                 <ul>
- <li>Provide ways for users to skip to important sections of your website. This will help users using screenreaders navigate your site easier and more efficiently.</li>
- <li>There is no real theming rule when it comes to styling your skip links, as such a lot of websites tend to hide them with <code>class=&quot;visuallyhidden&quot;</code> with a special focus attribute so that sighted keyboard only users are able to see them.
- <li>It is important to see that the link points to a valid HTML ID, as often this is overlooked in implementation.</li>
- <li>In this site we have used 'Jump to main content', but other valid examples of &quot;skip&quot; links include:
- <pre>Skip to content<br>Skip to main content<br>Skip to search<br>Skip to bottom navigation<br>Skip to main menu</pre></li>
+<li>Provide ways for users to skip to important sections of your website. This will help users using screenreaders navigate your site easier and more efficiently.</li>
+<li>There is no real theming rule when it comes to styling your skip links, as such a lot of websites tend to hide them with <code>class=&quot;visuallyhidden&quot;</code> with a special focus attribute so that sighted keyboard only users are able to see them.</li>
+<li>It is important to see that the link points to a valid HTML ID, as often this is overlooked in implementation.</li>
+<li>In this site we have used 'Jump to main content', but other valid examples of &quot;skip&quot; links include:</li>
 </ul>
+ <pre>Skip to content<br>Skip to main content<br>Skip to search<br>Skip to bottom navigation<br>Skip to main menu</pre>
```

Comment:
just want to leave a comment that it might be a good idea to revisit / talk about this later.